man/systemd.network.xml

```diff
@@ -1051,6 +1052,32 @@
       </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>[IPv6Token] Section Options</title>
+
+      <para>An <literal>[IPv6Token]</literal> section accepts the
+      following keys. Specify several <literal>[IPv6Token]</literal>
+      sections to configure several prefix matches. IPv6 tokens are
+      used to generate interface identifiers with IPv6 Stateless Address Autoconfiguration (SLAAC).
+      See <ulink url="https://tools.ietf.org/html/rfc7217">RFC 7217</ulink>.</para>
+
+      <variablelist class='network-directives'>
+        <varlistentry>
+          <term><varname>PrefixMatch=</varname></term>
+          <listitem>
+            <para> Specifies which prefix should be matched. This key is mandatory.</para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term><varname>TokenAddressGenerationType=</varname></term>
+          <listitem>
+            <para>Specifies the address generation type. Takes one of prefixstable or eui64.
+            This key is mandatory. </para>
+          </listitem>
```

Comment:
The same as in #13663, please make this into a single line:
```
IPv6Token=prefixstable:DEADBEEF0123231423432
IPv6Token=eui64:DEADBEEF0123231423432
```

Reply:
Now `[IPv6Token]` section has only `Token=` key. Let's merge this setting with `IPv6Token=` in `[Network]` section.

Reply:
There are multiple prefix matches to be done.